build-tools/packages/version-tools/src/internalVersionScheme.ts

```diff
@@ -435,5 +470,6 @@ export function detectInternalVersionConstraintType(range: string): "minor" | "p
 	const minor = bumpInternalVersion(minVer, "minor");
 
 	const maxSatisfying = semver.maxSatisfying([patch, minor], range);
+	// maxSatisfying could return null in case of exact version. Should that return as "minor"?
```

Comment:
That should probably return "exact", which would change the function signature - which is also fine.

Reply:
What should bumpRange do when it encounters and exact constraint? throw?